managers/asset-operations-manager.js

```diff
@@ -790,32 +790,89 @@ class AssetOperationsManager {
     }
 
     /**
-     * Retrieves the issuer of a specified asset for a given blockchain.
+     * Retrieves the issuer of a specified asset for a specified state index and a given blockchain.
      * @async
      * @param {string} UAL - The Universal Asset Locator of the asset.
+     * @param {string} stateIndex - The state index of the assertion we want to get issuer of.
      * @param {Object} [options={}] - Optional parameters for blockchain service.
      * @returns {Object} An object containing the UAL, issuer and operation status.
      */
-    async getIssuer(UAL, options = {}) {
+    async getStateIssuer(UAL, stateIndex, options = {}) {
         const blockchain = this.inputService.getBlockchain(options);
-        this.validationService.validateAssetGetIssuer(UAL, blockchain);
+        this.validationService.validateAssetGetStateIssuer(UAL, blockchain);
 
         const { tokenId } = resolveUAL(UAL);
-        const assertionIndex = 0;
-        const firstAssertionId = await this.blockchainService.getAssertionIdByIndex(
+
+        const assertionId = await this.blockchainService.getAssertionIdByIndex(
             tokenId,
-            assertionIndex,
+            stateIndex,
             blockchain,
         );
 
-        const issuer = await this.blockchainService.getAssertionIssuer(tokenId, firstAssertionId, assertionIndex, blockchain);
+        const issuer = await this.blockchainService.getAssertionIssuer(
+            tokenId,
+            assertionId,
+            stateIndex,
+            blockchain,
+        );
+        return {
+            UAL,
```

Comment:
It makes to return state as well imo, just so user has a possibility to check that it's the state he wanted